README.md

```diff
@@ -8,7 +8,7 @@
 
 # RouterOS API Client
 
-    composer require evilfreelancer/routeros-api-php
+    composer require proner/routeros-api-php
```

Comment:
Seems this should be changed to original group name.

Reply:
Sorry, i changed vendor name for my fork and forgeted back changed before pull request

Reply:
It's okay, just do not forget to fix it before merge :)